src/projections/healpix.cpp

```diff
@@ -622,6 +637,8 @@ PJ *PROJECTION(healpix) {
     P->opaque = Q;
     P->destructor = destructor;
 
+    Q->rot = pj_param(P->ctx, P->params,"drot").f * M_PI / 180.0;
```

Comment:
Improve readability a bit:

```suggestion
	double angle = pj_param(P->ctx, P->params,"drot").f;
    Q->rot = PJ_TORAD(angle);
```